ChaosMod/Effects/db/Player/PlayerZoomZoomCam.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+    Effect by Moxi, based on code from DrUnderscore & ProfessorBiddle
+*/
+
+#include <stdafx.h>
+static Camera zoomCamera = 0;
+float camZoom = 80.f;
+const float camZoomRate = 0.6f;
+int n = 0;
+
+static void UpdateCamera()
+{
+    auto coord = CAM::GET_GAMEPLAY_CAM_COORD();
```

Comment:
Please use the corresponding typename instead of „auto“
Namespace prefixes like „CAM::“ can be omitted

Reply:
I think it's okay now. When is a namespace prefix needed?

Reply:
Namespaces are needed if they are not included which they are via the „stdafx.h“ Header